app/services/locations/updater.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Locations
+  class Updater
+    YOI_NOMIS_AGENCIES = %w[WYI WNI CKI PRI FMI].freeze
+
+    def self.call
```

Comment:
rather than a separate process, could this be done as part of the rake task `reference_data:create_locations` - that gets run relatively regularly and creates / updates / disables locations

Reply:
I prefer to keep it as a separate process but will tack it onto the end of the create locations task too so that it's automatically invoked when importing new locations, just in case any of those happen to be a YOI.

Reply:
that sounds fine 👍